### PR TITLE
change favicon , contact us image and fonts

### PR DESCRIPTION
**Pull request by nisalrenuja**
- [X] Fixed #139

### CODE REVIEW

File: src/app/components/about/about.component.css [hunk 1]
```diff
@@ -37,6 +37,18 @@ a:hover {
...
+  height: 100%;
+  width: 80%;
+  transition: 1s ease;
+}
+.img-fluid_1 img:hover {
+  -webkit-transform: scale(0.8);
+  -ms-transform: scale(0.8);
+  transform: scale(0.8);
+  transition: 1s ease;
+}
```
Comment (Shehanka) [inline src/app/components/about/about.component.css:50]: change css class name img-fluid-MEANINFUL-NAME

**Review by Shehanka — CHANGES_REQUESTED** [review]

**Comment by chenukas** [comment]
@nisalrenuja it's better if you can enlarge the size of FOSS logo comparing to the black background of favicon. @Shehanka what do you think? Anyway the design is awesome.

**Comment by ebonynon** [comment]
Screenshot_2021-01-29_09-14-36

**Comment by ebonynon** [comment]
Screenshot_2021-01-29_09-15-02

**Comment by Shehanka** [comment]
> @nisalrenuja it's better if you can enlarge the size of FOSS logo comparing to the black background of favicon. @Shehanka what do you think? Anyway the design is awesome.

yeah, let's reduce the bg little bit

**Review by Shehanka — APPROVED** [review]
I gonna merge this. Send other changes with another PR

**Pull request merged**